site/docs/install-redhat.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: documentation
+title: Installing Bazel on Fedora and CentOS
+---
+# Install Bazel on Fedora and CentOS
+The Bazel team does not provide official packages for Fedora and CentOS.
+Vincent Batts [@vbatts](https://github.com/vbatts) generously maintains
+unofficial packages on [Fedora Copr].
+
+## Install Bazel on Fedora 25, 26
+
+You need to have `dnf` and the `copr` plugin installed.
+
+```bash
+dnf copr enable vbatts/bazel
+dnf install bazel
+```
+
+## Install Bazel on CentOS 7
+
+Download the corresponding `.repo` file from [Fedora Copr]
```

Comment:
Convert to link: [Fedora Copr](https://copr.fedorainfracloud.org/coprs/vbatts/bazel/)